### PR TITLE
Add multiple port forwarding with it

### PR DESCRIPTION
**Pull request by AndrewScibek**
Add punch it command.

open multiple tunnels using `type:port` 

Example command would be punch it http:80 https:443

### CODE REVIEW

File: cmd/it.go [hunk 1]
```diff
@@ -1,58 +1,87 @@
...
+	port        int
+	forwardType string
+}
+
 var itCmd = &cobra.Command{
 	Use:   "it [ports]",
 	Short: "Expose a web server on the ports you specify",
-	Long:  `To expose a web server on port 80 punch it 80`,
-	Args:  cobra.ExactArgs(1),
+	Long: `To expose a http web server on port 80 and https web server on port 443
```
Comment (stephenprater) [inline cmd/it.go]: I think these are confusing instructions, since the ports you provide are _your local ports_ and are unlikely to be 80/443.   I would reword this as:

```
To expose an http web server on your computer listening on port 5000 and an
https web server listening on port 5001 - punch it http:5000 https:5001
```

**Review by stephenprater — APPROVED** [review]

**Comment by AndrewScibek** [comment]
[Create Punch it command](https://trello.com/c/HJHRgKxA/273-create-punch-it-command)
